source/includes/examples/_find-the-center.html.md

```diff
@@ -0,0 +1,182 @@
+# Find the Center Example
+
+In this example, we'll walk you through the various statements that are part of the Find the Center project, including the simulator file and the Inkling file. Find the Middle was created as a very basic example of Inkling and how to connect to a simulator.
+
+The general premise of Find the Center is that the computer (via Inkling) takes a step (move) of -1, 0, or 1, and gets rewarded for staying in the center (1) for each round. The max is 2 and the min is 0.
+
+## Inkling File
+
+###### Schema
+
+```inkling
+schema GameState
+    Int32 value
+end
+```
+
+The `GameState` schema has one record, `value`, assigned the constrained type of Int32.
+
+```inkling
+schema PlayerMove
+    Int32{-1, 0, 1} delta
+end
+```
+
+The `PlayerMove` schema has one record, `delta`, assigned the constrained type of Int32 to allow for negative and positive values.
+
+```inkling
+schema SimConfig
+    Int32 dummy
+end
+```
+
+The `SimConfig` schema has one record, `dummy`, because there is no configuration needed in this particular example.
+
+###### Concept
+
+```inkling
+concept find_the_center
+    is classifier
+    predicts (PlayerMove)
+    follows input(GameState)
+    feeds output
+end
+```
+
+This concept is named `find_the_center`, and it takes input from the simulator about the state of the game (`GameState` schema). It outputs to the `PlayerMove` schema. This is the AI's next move in the simulation.
+
+###### Simulator
+
+```inkling
+simulator find_the_center_sim(SimConfig)
+    action (PlayerMove)
+    state (GameState)
+end
+```
+
+The `simulator` is called `find_the_center_sim` (shown in #simulator-file) and takes the schema input of `SimConfig` (even though it isn't configuring anything, it's required by the simulator). The simulator is going to output the `action` and the `state` of the simulator which is sent to the lesson.
+
+###### Curriculum
+
+```inkling
+curriculum find_the_center_curriculum
+    train find_the_center
+    with simulator find_the_center_sim
+    objective time_at_goal
+        lesson seek_center
+            configure
+                constrain dummy with Int32{-1}
+            until
+                maximize time_at_goal
+end
+```
+
+The curriculum is named `find_the_center_curriculum`, and it trains the `find_the_center` concept using the `find_the_center_sim`. This curriculum contains one lesson, called `seek_center`. It configures the simulation, by setting a number of constraints for the state of the simulator.
+
+The lesson trains until the AI has maximized the objective `time_at_goal`.
+
+
+## Simulator File
+
+```python
+import bonsai
+import sys
+from bonsai.simulator import SimState
+from random import randint
+
+""" If you would like to debug your code add this back in.
+def debug(*args):
```

Comment:
*args should work fine in python 2 (I think it worked in python 1 as well)...  but the print statement changes from 2 to 3.  to use python 3's print function from python 2, you must put `from __future__ import print_function` as the first line in the python file.